pkg/operator/k8sutil/node.go

```diff
@@ -306,8 +306,8 @@ func NodeIsInRookNodeList(targetNodeName string, rookNodes []rookalpha.Node) boo
 	return false
 }
 
-// AddNodeAffinity will return v1.NodeAffinity or error
-func AddNodeAffinity(nodeAffinity string) (*v1.NodeAffinity, error) {
+// GetNodeAffinity will return v1.NodeAffinity or error
+func GetNodeAffinity(nodeAffinity string) (*v1.NodeAffinity, error) {
```

Comment:
What about renaming to `GenerateNodeAffinity()`? Neither Add nor Get seem correct.

Reply:
sure, looks good